__tests__/components/search/GroupFilter.test.js

```diff
@@ -97,7 +97,7 @@ describe("<GroupFilter />", () => {
     })
   })
 
-  it("allows selecting only", async () => {
+  it("allows selecting / deselecting all", async () => {
```

Comment:
wondering if you want to test this functionality one for the SearchFilter component, rather than twice in these tests.  It would remove some duplication, no?